tox.ini

```diff
@@ -27,13 +27,9 @@ commands =
 
 [testenv:docs]
 basepython = python3.9
-# This is not optimal, will require more organization of Metal project.
-# Or, sloppy.... put all the packages from requirement.txt into requirement-dev.txt.
-skip_install = true
```

Comment:
I couldn't figure out why this was being done. Based on looking at `setup.py`, there's nothing fancy we're doing to build the package like using native extensions. 

Because `setup.py` sets `install_requires` to the value of `requirements.txt`, we will install `requirements.txt` like before, and now also `qiskit_metal` itself.